aptos-move/aptos-workspace-server/src/main.rs

```diff
@@ -99,66 +115,112 @@ async fn spawn_node(test_dir: &Path) -> Result<()> {
         }
     });
 
-    let api_port = api_port_rx.await?;
-    let indexer_grpc_port = indexer_grpc_port_rx.await?;
+    let fut_api = async move {
+        let api_port = api_port_rx.await?;
 
-    let api_health_checker = HealthChecker::NodeApi(
-        Url::parse(&format!(
-            "http://{}:{}",
-            node_config.api.address.ip(),
-            api_port
-        ))
-        .unwrap(),
-    );
-    let indexer_grpc_health_checker = HealthChecker::DataServiceGrpc(
-        Url::parse(&format!(
-            "http://{}:{}",
-            node_config.indexer_grpc.address.ip(),
-            indexer_grpc_port
-        ))
-        .unwrap(),
-    );
+        let api_health_checker = HealthChecker::NodeApi(
+            Url::parse(&format!("http://{}:{}", IP_LOCAL_HOST, api_port)).unwrap(),
+        );
+        api_health_checker.wait(None).await?;
 
-    api_health_checker.wait(None).await?;
-    eprintln!(
-        "Node API is ready. Endpoint: http://127.0.0.1:{}/",
-        api_port
-    );
+        eprintln!(
```

Comment:
please dont use `eprintln!` for non-errors
https://github.com/aptos-labs/aptos-core/pull/15005